src/ploigos_step_runner/step_implementers/audit_attestation/opa.py

```diff
@@ -0,0 +1,184 @@
+"""`StepImplementer` for the `opa` step using Open Policy Agent.
+
+Step Configuration
+------------------
+Step configuration expected as input to this step.
+Could come from:
+
+  * static configuration
+  * runtime configuration
+  * previous step results
+
+Configuration Key       | Required? | Default | Description
+-------------------     |-----------|---------|-----------
+`workflow-policy-uri`   | Yes       |         | URI to policy used to audit attestations
+
+Result Artifacts
+----------------
+Results artifacts output by this step.
+
+Result Artifact Key | Description
+--------------------|------------
+`audit-results`     | Results of opa audit of workflow attestations
+"""
+
+import sys
+from io import StringIO
+import sh
+from ploigos_step_runner import StepImplementer, StepResult
+from ploigos_step_runner.utils.file import download_source_to_destination
+from ploigos_step_runner.utils.io import \
+    create_sh_redirect_to_multiple_streams_fn_callback
+
+
+DEFAULT_CONFIG = {}
+
+REQUIRED_CONFIG_OR_PREVIOUS_STEP_RESULT_ARTIFACT_KEYS = [
+    'workflow-policy-uri'
+]
+class OpenPolicyAgent(StepImplementer):  # pylint: disable=too-few-public-methods
+    """`StepImplementer` for the generic Rekor class.
+    """
+
+    DEFAULT_WORKFLOW_POLICY_QUERY = "data.workflowResult.passAll"
+    DEFAULT_WORKFLOW_POLICY_DATA_QUERY = "data"
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        workflow_result,
+        parent_work_dir_path,
+        config,
+        environment=None
+    ):
+
+        super().__init__(
+            workflow_result=workflow_result,
+            parent_work_dir_path=parent_work_dir_path,
+            config=config,
+            environment=environment
+        )
```

Comment:
Fix up the indentation.

```suggestion
    super().__init__(
         workflow_result=workflow_result,
         parent_work_dir_path=parent_work_dir_path,
         config=config,
         environment=environment
    )
```